addons/web/views/report_templates.xml

```diff
@@ -146,16 +146,16 @@
                            <span>INV/2020/07/0003</span>
                        </h2>
                        <div id="informations" class="row mt32 mb32">
-                           <div class="col-auto mw-100 mb-2" name="invoice_date">
+                           <div class="col-auto mw-100 mb-4" name="invoice_date">
```

Comment:
I still see a lot of differences with the classes used compared to the real template
https://github.com/odoo/odoo/blob/d53d0222a8872be8934f225c8515a3bc9d7fdc49/addons/account/views/report_invoice.xml#L63-L64

Actually, on this line it is a `mb-2` like before